### PR TITLE
checking global classroom camera register, checking roles on users

### PR DESCRIPTION
**Pull request by marknutter**

### CODE REVIEW

File: app/api/camera_data.py [hunk 1]
```diff
@@ -28,17 +28,25 @@ def camera_data_index():
...
-    classroom_ref = firebase.db.document(path)
-    doc = classroom_ref.get()
-    camera_buckets = doc.to_dict().get("camera_buckets")
-    if camera_buckets:
-      permitted_buckets.extend(doc.to_dict().get("camera_buckets"))
+  firebase = current_app.config.get("FIREBASE_SERVICE")
+  classroom_ref = firebase.db.collection('cameras')
+  docs = classroom_ref.get()
+  
+  for c in user.get("accessible_classroom_ids"):
```
Comment (ps2) [inline app/api/camera_data.py]: c does not appear to be used outside of a print statement below

File: app/api/camera_data.py [hunk 2]
```diff
@@ -28,17 +28,25 @@ def camera_data_index():
...
+    for doc in docs:
+      camera = doc.to_dict()
+      print "c: %s, classroomId: %s" % (c, camera.get("classroomId"))
+      if ('admin' in user.get("roles") or 'network_admin' in user.get("roles")) and camera.get("bucketName") not in permitted_buckets:
```
Comment (ps2) [inline app/api/camera_data.py]: I don't think this is right.

A network admin can access all classrooms. An admin should still only be able to access the classrooms listed in their their accessible_classroom_ids list.

Comment (marknutter) [reply app/api/camera_data.py]: Right, good call.

File: app/api/camera_data.py (hunk 1 above)
Comment (marknutter) [reply app/api/camera_data.py]: Yes, I noticed this as I started working on getting the locations viz wired up on another branch. I'll merge that one into this branch which has this and other issues sorted out.

**Comment by marknutter** [comment]
I merged the camera-segment-builder-ux-improvements branch I was working on which contains fixes for the issues you identified @ps2

**Pull request merged**